pkg/microservice/aslan/core/common/service/workflowcontroller/jobcontroller/job.go

```diff
@@ -87,7 +87,7 @@ func runJob(ctx context.Context, job *commonmodels.JobTask, workflowCtx *commonm
 		}
 		return true
 	})
-	job.Status = config.StatusRunning
+	job.Status = config.StatusPrepare
```

Comment:
if we set status here, every job status will be StatusPrepare，what if other job did not set job status to running?